components/settings/NetPyNETabs.js

```diff
@@ -63,9 +64,9 @@ export default class NetPyNETabs extends React.Component {
                 targetOrigin={{ horizontal: "right", vertical: "top" }}
                 anchorOrigin={{ horizontal: "right", vertical: "bottom" }}
             >
-                <MenuItem primaryText="Create Network" value="Create Network" />
-                <MenuItem primaryText="Create and Simulate Network" value="Create and Simulate Network" />
-                <MenuItem primaryText="Explore Existing Network" value="Explore Existing Network" />
+                <MenuItem id="transit_not" primaryText="Create Network" value="Create Network" />
```

Comment:
should be transit_net?

Reply:
btw, what is the rule regarding id naming? camelcase or underscore?

Reply:
it would be exp (explore), cre (create), sim(simulate) .  which are the primaryText in the appbar dropdown menus

Reply:
so... are you changing it? Try to use the same rule for all the ids please: either camelcase or underscore. In general, it looks like you are using camelcase...